backend/database_handler/validators_registry.py

```diff
@@ -56,6 +56,11 @@ def create_validator(self, validator: Validator) -> dict:
         self.session.add(_to_db_model(validator))
         return self.get_validator(validator.address)
 
+    def register_leader_as_validator(self, validator: Validator) -> dict:
+        if not self.validator_exists(validator.address):
+            self.create_validator(validator)
+        return self.get_validator(validator.address)
+
```

Comment:
We are not registering/deregistering at the database level. Instead, we pick them up at runtime in the consensus algorithm